### DISCUSSION TITLE
Display routes at bus stop

### DISCUSSION THREAD

**Discussion by nlehuby** [organicmaps/organicmaps#1215] [Ideas]
When a bus stop (`highway`=`bus_stop`) is part of some bus routes (relation with `route`=`bus`), it could be great to display these routes when you click on the bus stop details (for instance the name of the relation)
It will help to know which bus you can get in at this stop.

**Comment by nlehuby** [comment]
same as #2175

**Comment by rtsisyk** [comment]
Please continue in https://github.com/organicmaps/organicmaps/issues/2502
